lib/sensors/low_power_bms.cpp

```diff
@@ -0,0 +1,56 @@
+#include "low_power_bms.hpp"
+
+namespace hyped::sensors {
+
+std::optional<LowPowerBMS> LowPowerBMS::create(core::ILogger &logger,
+                                               std::shared_ptr<io::II2c> i2c,
+                                               const std::uint8_t device_address)
+{
+  if (device_address != kDefaultLpBmsAddress) {
```

Comment:
if this can only take one address then don't take a device address, if it can be one of a selection check if it's any of them, if it can be any address then this isn't needed